nfv_tempest_plugin/tests/scenario/dcb_manager.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2018 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+import yaml
+from nfv_tempest_plugin.tests.common import shell_utilities as shell_utils
+from oslo_log import log
+from tempest import config
+
+CONF = config.CONF
+LOG = log.getLogger('{} [-] nfv_plugin_test'.format(__name__))
+
+class DCBManager(object):
+
+    def create_and_apply_dcb_config(self, target_interface, dut_hypervisor):
```

Comment:
why should we add dcb config in testcase?
is this something we have already in our deployment instead of modifying config in testcase?

Reply:
I will try to move to day2 folder. Since its day 2 configuration which we are doing.

Reply:
Changes updated. Kindly check.